probe/kubernetes/kubelet_test.go

```diff
@@ -0,0 +1,73 @@
+package kubernetes_test
```

Comment:
Why are we putting tests in a different package to the code being tested, and hence have to export functions that otherwise would be local? I don't think that's The Go Way.